bayesian-bandit.js

```diff
@@ -70,8 +78,14 @@
 
     this.arms = []
 
-    for(var a = 0; a < (options || {}).numberOfArms; a++) {
-      this.arms.push(new Arm())
+    if (!Array.isArray(options)) {
+      for (var a = 0; a < (options || {}).numberOfArms; a++) {
+        this.arms.push(new Arm())
+      }
+    } else {
+      for (var i = 0; i < options.length; i++) {
+        this.arms.push(new Arm(options[i]))
+      }
```

Comment:
Maybe we could accept a parameter called `arms` which is an array?  I think it makes the constructor invocation a little easier to read if the optional parameter is labeled.  What do you think?